src/rest/api.ts

```diff
@@ -451,6 +451,16 @@ async function fetchTaxRate(): Promise<TaxRateResponse> {
   }
 }
 
+async function fetchRelayerFee(): Promise<any> {
+  const ics20Contract = new CwIcs20LatestQueryClient(window.client, process.env.REACT_APP_IBC_WASM_CONTRACT);
+  try {
+    const { relayer_fees } = await ics20Contract.config();
```

Comment:
Why not use `camelCase` like other variables ?

Reply:
It is a key in the object so i used it always
![Screen Shot 2023-10-12 at 16 28 37](https://github.com/oraichain/oraiswap-frontend/assets/54303398/fb797ff6-1257-460e-b49b-d8aa974b997d)

Reply:
Then I would success to contract object with `Ics2Contract` type , then return properly with `Ics2Contract.relayer_fees`. It makes context clearer. 

But not very critical one. You can add comment `//REFACTOR:` for managing refactor list later.